pysmt/operators.py

```diff
@@ -55,7 +55,11 @@
 BV_ASHR,                                    # Arithmetic shift right (47)
 ARRAY_SELECT,                               # Array Select (48)
 ARRAY_STORE,                                # Array Store (49)
-ARRAY_VALUE                                 # Array Value (50)
+ARRAY_VALUE,                                # Array Value (50)
+
+DIV,                                        # Arithmetic Division (51)
```

Comment:
Is there a reason to explicitly represent Div(a, b) internally instead of using Times(a, Pow(b, -1))? This would reduce the number of operators and simplify the walkers, so unless there is an efficiency or technical reason for having the explicit div I would remove it.